src/soc/ibm/power9/Kconfig

```diff
@@ -22,4 +22,22 @@ config SKIP_INITIAL_ECC_SCRUB
 	  Selecting this option almost halves the time needed for memory
 	  initialization (istep 14.1).
 
+if BOARD_RAPTOR_CS_TALOS_2
+
+config TALOS_2_INFINEON_TPM_1
```

Comment:
SoC directory should be clear of mainboard names, if possible. I assume this is done to hide TPM option for QEMU?